lessons/18-js-introduction/about.js

```diff
@@ -0,0 +1,12 @@
+<html>
+    <head>
+        <title>About Page</title>
+        <body>
+            <script>
+            //Show a popup Hello World
+            alert("This is the about page")
+        
+            </script>
+        </body>
+    </head>
+</html>
```

Comment:
Careful here, this is a `.js` file not a `.html` so it can only contain javascript, not html